__main__.py

```diff
@@ -90,18 +158,19 @@ def main() -> None:
     new = get_current_state()
 
     diff = old ^ new
-    delta = pretty_delta(old.date, new.date)
+    delta = pretty_delta(new.time - old.time)
 
     if not diff:
         dprint(f"&cNo updates detected in the last {delta}.")
-        sys.exit()
+        return
```

Comment:
Using `return` instead of `sys.exit()` here not only makes it match the case where `diff` is truthy, but it also results in better output when running the code in interactive mode, with `python -i .`.

With `sys.exit()`, it'd print a traceback when there were no updates.

I've also added another empty `return` to the end of this function because IIRC PEP8 says that you should either have no `return`, or always have `return` for every possibility.